main/settings.py

```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+import django_heroku
```

Comment:
We don't require this additional package because I have deployed many sites without this.
Do you think this is required in deployment?

Reply:
@codesankalp I have not much experience with django, so can't say. But yeah, I was following heroku official [docs](https://devcenter.heroku.com/articles/django-app-configuration) and [this](https://medium.com/analytics-vidhya/step-by-step-guide-for-deploying-a-django-application-using-heroku-for-free-cc0cb5901f8a) medium article which mentioned the use of this package.

Reply:
Leave it then but make other changes.